ludwig/features/binary_feature.py

```diff
@@ -53,6 +53,63 @@
 logger = logging.getLogger(__name__)
 
 
+class _BinaryPreprocessing(torch.nn.Module):
+    def __init__(self, metadata: Dict[str, Any]):
+        super().__init__()
+        str2bool = metadata.get("str2bool")
+        self.str2bool = str2bool or {v: True for v in strings_utils.BOOL_TRUE_STRS}
+        self.should_lower = str2bool is None
+
+    def forward(self, v: Union[List[str], torch.Tensor]):
+        if isinstance(v, torch.Tensor):
+            return v.to(dtype=torch.bool)
+
+        v = [s.strip() for s in v]
+        if self.should_lower:
+            v = [s.lower() for s in v]
+        indices = [self.str2bool.get(s, False) for s in v]
+        return torch.tensor(indices, dtype=torch.bool)
+
+
+class _BinaryPostprocessing(torch.nn.Module):
+    def __init__(self, metadata: Dict[str, Any]):
+        super().__init__()
+        bool2str = metadata.get("bool2str")
+        self.bool2str = {i: v for i, v in enumerate(bool2str)} if bool2str is not None else None
+        self.predictions_key = PREDICTIONS
+        self.probabilities_key = PROBABILITIES
+
+    def forward(self, preds: Dict[str, torch.Tensor]) -> Dict[str, Any]:
+        predictions = preds[self.predictions_key]
+        if self.bool2str is not None:
+            predictions = predictions.to(dtype=torch.int32)
+            predictions = [self.bool2str.get(pred, self.bool2str[0]) for pred in predictions]
+
+        probs = preds[self.probabilities_key]
+        probs = torch.dstack(1 - probs, probs)
+
+        return {
+            self.predictions_key: predictions,
```

Comment:
Similar question here. It seems simpler to reference global constants directly instead of first setting them as class attributes, i.e.:

```
return {
    PREDICTIONS: predictions,
    PROBABILITIES: probs,
}
```